feature-detects/webrtc/datachannel.js

```diff
@@ -19,8 +19,12 @@ define(['Modernizr', 'prefixed', 'domPrefixesAll', 'test/webrtc/peerconnection']
     for (var i = 0, len = domPrefixesAll.length; i < len; i++) {
       var PeerConnectionConstructor = window[domPrefixesAll[i] + 'RTCPeerConnection'];
       if (PeerConnectionConstructor) {
-        var peerConnection = new PeerConnectionConstructor(null);
-        return 'createDataChannel' in peerConnection;
+        // PS4 throws "Error creating RTCPeerConnection" when attempting to create an instance of webkitRTCPeerConnection
```

Comment:
Seems to be more than the PS4 error, maybe a better comment could be `Wrapped in a try catch to avoid "Error creating RTCPeerConnection" #2599 & #2221`. What do you think? 🤔😊

Reply:
That's better!